Resources/Private/Scss/_news.scss

```diff
@@ -114,5 +100,23 @@
         @include media-s {
             @include span-s-6;
         }
+        font-size: $large-font-size-l;
+        line-height: $large-line-height-l;
+        font-weight: bold;
+        align-content: flex-start;
+
+        .title {
+            @include media-l {
+                @include span-l-8;
```

Comment:
related to my previous comment: this could be just "span-8" and what that translates to could be dealt with in @mixin span($value) { … }